manifests/c/CodecGuide/K-LiteCodecPack/Standard/17.3.5/CodecGuide.K-LiteCodecPack.Standard.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CodecGuide.K-LiteCodecPack.Standard
+PackageVersion: 17.3.5
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files3.codecguide.com/K-Lite_Codec_Pack_1735_Standard.exe
+  InstallerSha256: 2acd40f6d12fe2bca7d75451a31c85e1d4ce9cb87fb88cdca533e4248625e59f
+  AppsAndFeaturesEntries:
+  - DisplayName: K-Lite Codec Pack 17.3.0 Standard

```

```suggestion
  - DisplayName: K-Lite Codec Pack 17.3.5 Standard
```
Must match Control Panel